about.html

```diff
@@ -0,0 +1,161 @@
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="style2.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <!-- mobile menu: only visible on mobile -->
+      <div class="header-sect">
+        <span class="clas"></span>
+        <div id="mob-nav">
+          <ul class="menu-1">
+            <li>
+              <a href="index.html">Home</a>
+            </li>
+            <li>
+              <a href="#">Program</a>
+            </li>
+            <li>
+              <a href="#">Join</a>
+            </li>
+            <li>
+              <a href="#">Sponsor</a>
+            </li>
+          </ul>
+        </div>
+        <span class="clas"></span>
+      </div>
+      <!-- Desktop menu: only visible on desktop -->
+      <nav>
+        <div class="media">
+          <ul>
+            <li>
+              <a><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a>English</a></li>
+            <li><a>My page</a></li>
+            <li><a>Logout</a></li>
+          </ul>
+        </div>
+        <div class="navba">
+          <a href="index.html"><img src="images/Capture.png" class="navImage" alt="" /></a>
+          <ul>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Program</a></li>
+            <li><button type="button">sports Campaign</button></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section class="ban">
+        <h1 class="margin-left">the sports summit  program</h1>
+        <div class="ban-summary move white-bg">
+          <p>
+            Like all good things in this world,
+             tournaments cost money to create.
+              Before spending any money on the event, 
+              a budgetary plan and goal needs to be set.
+               Nevertheless, creating a budget for a youth football
+                tournament can be difficult, 
+            especially for those who are hosting for the first time.
+           
+          </p>
+        </div>
+        <p class="place side">
+          Please contact per Email for any further questions about sports summit
+          program 2021 <br />
+          <span>rochidzwasports@gmail.com</span>
+        </p>
+      </section>
```

Comment:
- [ ] Kindly center the elements as seen in the original design. You can make use of Flexbox.

| Current Design | Original Design |
| --- | --- |
|![image](https://user-images.githubusercontent.com/4247740/164861450-fe73cc71-c2b1-4b9b-9712-f95a850e0c28.png) | ![image](https://user-images.githubusercontent.com/4247740/164861564-9ff24d2f-f8a1-4b67-ab44-80e777ec5238.png) |